### PR TITLE
#80: implement support for renaming group

### LINKED ISSUE #80: Renaming group
We have a need for renaming groups in Keycloak. This can be done through UI.

From what I can see, this works through same APi as `updateGroup`, where the only change would be the `name`.

This is suggested API (I'll open PR soon):
```
id: rename-group
author: mfolnovic
changes: 
  - renameGroup:
      realm: testRealm
      name: GROUP_A
      newName: NEW_GROUP_A
```

I'm also open to reusing `updateGroup` action:

```
id: rename-group
author: mfolnovic
changes: 
  - updateGroup:
      realm: testRealm
      name: GROUP_A
      newName: NEW_GROUP_A
```

I'm also open to any other suggestion, such as better property name than `newName`. :slightly_smiling_face:

### PR DESCRIPTION
**Pull request by mfolnovic**
resolves #80

### CODE REVIEW

File: src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/RenameGroupAction.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class RenameGroupAction(
+    realm: String? = null,
+    private val name: String,
+    private val newName: String) : Action(realm) {
+
+    private lateinit var group: Group
+
+    private fun updateGroup() = UpdateGroup(newName, group.path, group.attributes,
+        //FIXME
```
Comment (klg71) [inline src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/RenameGroupAction.kt:18]: Could you elaborate how a fix could be implemented here?

**Comment by klg71** [comment]
Hey @mfolnovic this Pull Request looks good in my eyes. I would be glad if you could comment on the `FIXME` issue. After that I see no problem in merging and releasing it :)

File: src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/RenameGroupAction.kt (hunk 1 above)
Comment (mfolnovic) [reply src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/RenameGroupAction.kt:18]: this is pretty much copy paste from https://github.com/mayope/keycloakmigration/blob/master/src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/UpdateGroupAction.kt 

that's where suggestion to expand `updateGroup` comes from :)

Comment (klg71) [reply src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/RenameGroupAction.kt:18]: Ok than it is obviously on me to fix this ^^

**Pull request merged**